ui/src/app/components/workspace/workspace.component.ts

```diff
@@ -16,6 +16,8 @@ import { extractSourceDbName } from 'src/app/utils/utils'
 import { ClickEventService } from 'src/app/services/click-event/click-event.service'
 import IViewAssesmentData from 'src/app/model/view-assesment'
 import IDbConfig from 'src/app/model/db-config'
+import { InfodialogComponent } from '../infodialog/infodialog.component'
```

Comment:
Rename to something more meaningful -  E.g `ConfirmSchemaConversionComponent`

Reply:
It is an existing component and same component is reused in multiple places therefore a generic name makes more sense IMO